
Older gcc (<3.4) are not able to honor the .rodata .section specification
correctly:

../sysdeps/i386/i686/memcmp.S: Assembler messages: 
../sysdeps/i386/i686/memcmp.S:390: Error: can't resolve `.rodata'  

  Rene Rebe <rene@exactcode.de>

--- ./sysdeps/i386/i686/memcmp.S~	2004-04-02 01:28:37.000000000 +0200
+++ ./sysdeps/i386/i686/memcmp.S	2004-05-22 21:27:39.000000000 +0200
@@ -351,7 +351,6 @@
 	popl	%esi
 	RETURN
 
-	.section	.rodata
 	ALIGN (2)
 L(table_32bytes) :
 	.long	L(0bytes) - . + 0x0
